common-crafting.lic

```diff
@@ -142,4 +142,13 @@ module DRCC
     end
     true
   end
+  
+  def logbook_item(logbook,noun)
+    DRC.bput("get my #{logbook} logbook", "You get")
+    case bput("bundle my #{noun} with my logbook", 'You notate the', 'This work order has expired', 'The work order requires items of a higher quality')
+    when 'This work order has expired', 'The work order requires items of a higher quality'
+      dispose_trash(noun)
+    end
+    DRC.bput("stow #{logbook} logbook", 'You put')
```

Comment:
What about something like this: It will grab the logbook from wherever it is on the person, but stow it in the crafting container